resharper/resharper-unity/src/Feature/Caches/DeferredCacheController.cs

```diff
@@ -167,6 +167,7 @@ public bool IsProcessingFiles()
                 {
                     // assertion is incorrect, because file could be removed while we have read lock.
                     // Assertion.Assert(!myDeferredHelperCache.FilesToProcess.Contains(psiSourceFile), "!myDeferredHelperCache.FilesToProcess.Contains(psiSourceFile), removed");
+                    myLogger.Verbose("Remove from processing removed file: {0}", psiSourceFile.GetPersistentIdForLogging());
```

Comment:
Can the message be updated to better reflect that the file has been (unexpectedly) deleted, please?